azure-pipelines.yml

```
@@ -3,7 +3,7 @@ resources:
     - repository: templates
       type: github
       name: NSLS-II/profile-collection-ci
-      ref: refs/heads/main
+      ref: refs/heads/enh-better-message-on-download-failure
```

https://github.com/NSLS-II/profile-collection-ci/pull/29 has just been merged. Please update the config as suggested below:
```suggestion
      ref: refs/heads/main
```